src/backend/Backend.ts

```diff
@@ -20,7 +21,7 @@ export default class Backend {
   private fileRepository: DBRepository<IFile>;
   private tagRepository: DBRepository<ITag>;
   private locationRepository: DBRepository<ILocation>;
-  private db: Dexie.Dexie;
```

Comment:
What is the difference?

Reply:
Nothing, the other type just didn't exist anymore after the version bump